server/lib/allowed-features.ts

```diff
@@ -38,22 +92,33 @@ export const hasOptedInForFeature = (collective, feature): boolean => {
 };
 
 /**
- * Check if the given feature is activated for collective.
+ * If a given feature is allowed for the collective type, check if it is activated for collective.
  */
-const hasFeature = (collective, feature: FEATURE): boolean => {
+export const hasFeature = (collective, feature: FEATURE): boolean => {
   if (!collective) {
     return false;
   }
 
-  // Check type
-  if (!isFeatureAllowedForCollectiveType(collective.type, feature)) {
```

Comment:
No longer checking the type? This is breaking for https://github.com/opencollective/opencollective-api/blob/d64f4f69eb8ede92ad3cf16ba83f1b811f574177/server/graphql/common/conversations.ts#L36, as people will be able to submit conversations to user profiles for example. I think we should keep this check.